### PR TITLE
Docteam membership update

### PR DESCRIPTION
**Pull request by henrik-ch**
This PR is not ready to be merged yet.

Made in cooperation with @infinisil at the latest docteam meeting - and these meeting notes were the basis for it.
https://discourse.nixos.org/t/2023-10-19-documentation-team-meeting-notes-87/34349

I will create this PR as a draft, ask for a 4 eye review and tag all involved users so that they get a chance to confirm or make changes.

### CODE REVIEW

**Comment by henrik-ch** [comment]
tagging users for commenting/confirming their membership status, please reply with 👍 if it looks good in your case, and otherwise any modifications that are needed.

@fricklerhandwerk 
@infinisil 
@olafklingt 
@zmitchell 
@asymmetric 
@proofconstruction 
@roberth 
@alejandrosame 
@yukiisbored

File: community/teams/documentation.tt [hunk 1]
```diff
@@ -15,16 +15,47 @@
 <section class="governance-team">
 
   <aside>
-    <h2>Members</h2>
 
+    <h2>Leader</h2>
     <ul>
       <li>
         Valentin Gagarin (<a href="https://discourse.nixos.org/u/fricklerhandwerk">@fricklerhandwerk</a>)<br/>
-        Nix documentarian, sponsored by <a href="https://antithesis.com/">Antithesis</a> via <a href="https://tweag.io">Tweag</a>
```
Comment (fricklerhandwerk) [inline community/teams/documentation.tt:23]: I think we should keep information on affiliations, as that's easy to find as opposed to PR annotations, and I wouldn't know a more suitable and easier to maintain places.

File: community/teams/documentation.tt [hunk 2]
```diff
@@ -15,16 +15,47 @@
 <section class="governance-team">
 
   <aside>
-    <h2>Members</h2>
 
+    <h2>Leader</h2>
```
Comment (fricklerhandwerk) [inline community/teams/documentation.tt:19]:
```suggestion
    <h2>Team Lead</h2>
```

Leader evokes weird associations for me.

File: community/teams/documentation.tt (hunk 1 above)
Comment (infinisil) [reply community/teams/documentation.tt:23]: Sounds good too, I suggested to remove it originally to keep it simple and the same for everybody, but I guess people can always PR to add their affiliations later if desired.

Comment (zmitchell) [reply community/teams/documentation.tt:23]: When you say “affiliations” are you referring to employers or sponsors? I have an employer (flox) but they aren’t _sponsoring_ my work. Should they be listed?

Comment (infinisil) [reply community/teams/documentation.tt:23]: I think this should be up to you :)

Comment (infinisil) [reply community/teams/documentation.tt:23]: @henrik-ch Let's keep the existing affiliations then, can you update the PR as such?

File: community/teams/documentation.tt [hunk 3]
```diff
@@ -15,16 +15,47 @@
 <section class="governance-team">
 
   <aside>
-    <h2>Members</h2>
 
+    <h2>Team Lead</h2>
     <ul>
       <li>
         Valentin Gagarin (<a href="https://discourse.nixos.org/u/fricklerhandwerk">@fricklerhandwerk</a>)<br/>
```
Comment (fricklerhandwerk) [inline community/teams/documentation.tt:22]:
```suggestion
        Valentin Gagarin (<a href="https://discourse.nixos.org/u/fricklerhandwerk">@fricklerhandwerk</a>)<br/>
        Nix documentarian, sponsored by <a href="https://antithesis.com/">Antithesis</a> via <a href="https://tweag.io">Tweag</a>
```

File: community/teams/documentation.tt [hunk 4]
```diff
@@ -15,16 +15,48 @@
...
         Nix documentarian, sponsored by <a href="https://antithesis.com/">Antithesis</a> via <a href="https://tweag.io">Tweag</a>
       </li>
+    </ul>
+
+
+    <h2>Members</h2>
+
+    <ul>
       <li>
         Silvan Mosberger (<a href="https://discourse.nixos.org/u/infinisil">@infinisil</a>)<br/>
```
Comment (infinisil) [inline community/teams/documentation.tt:32]:
```suggestion
        Silvan Mosberger (<a href="https://discourse.nixos.org/u/infinisil">@infinisil</a>)<br/>
        <a href="https://nixpkgs.org">Nixpkgs</a> maintainer, sponsored by <a href="https://tweag.io">Tweag</a>
```

File: community/teams/documentation.tt [hunk 5]
```diff
@@ -15,16 +15,48 @@
...
+        wamirez (<a href="https://discourse.nixos.org/u/wamirez">@wamirez</a>)<br/>
+      </li>
+      <li>
+        yukiisbored (<a href="https://github.com/yukiisbored">@yukiisbored</a>)<br/>
+      </li>
+      <li>
+        proofconstruction (<a href="https://discourse.nixos.org/u/proofconstruction">@proofconstruction</a>)<br/>
+      </li>
+      <li>
+        roberth (<a href="https://discourse.nixos.org/u/roberth">@roberth</a>)<br/>
```
Comment (roberth) [inline community/teams/documentation.tt:56]:
```suggestion
        Robert Hensing (<a href="https://discourse.nixos.org/u/roberth">@roberth</a>)<br/>
```

**Review by fricklerhandwerk — APPROVED** [review]

**Pull request merged**